sync-online/service-worker.js

```diff
@@ -0,0 +1,147 @@
+// [Working example](/serviceworker-cookbook/sync-online/).
+
+var CACHE_NAME = 'dependencies-cache';
+
+// Files required to make this app work offline
+var REQUIRED_FILES = [
+  '/',
+  '/index.html',
```

Comment:
Can remove this once https://github.com/mozilla/serviceworker-cookbook/pull/126 is merged.